blocks/header-nav-chain-block/chains/header-nav-chain-block/useDebounce.js

```diff
@@ -0,0 +1,84 @@
+// https://github.com/xnimorz/use-debounce
```

Comment:
can we use useDebounce package rather than copy-paste their code? if we use the package as a dependency, then we can get updates to it potentially

Reply:
yes sure, I don't added it as dependency because i thought we want to minimize external dependencies.

Reply:
I'm not sure what you mean. If you need to use this package, include it as a dependency.

Reply:
I think he is referring to the fact that we want to keep each block's external dependencies (non-wpmedia node modules) at a minimum, but I think in this scenario it makes sense to add `use-debounce` to `header-nav-chain-block`